i3: extend module

uvNikita opened the pull request:
New options:
  i3.config.keycodebindings
  i3.config.window.commands
  i3.config.window.hideEdgeBorders
  i3.config.focus.mouseWarping

rycee commented on modules/misc/news.nix:
```diff
@@ -462,6 +462,18 @@ in
+        time = "2017-11-17T10:36:10+00:00";
+        condition = config.xsession.windowManager.i3.enable;
+        message = ''
+          i3 window Manager module has been extended with the following options:
```
Nicer to start with a capital letter. Maybe "The i3 window manager module …"? Btw, the m in Manager seems randomly capitalized :-)

rycee commented:
LGTM 👍

uvNikita commented:
Fixed suggestions and rebased into master: https://github.com/rycee/home-manager/commit/177565567e793e5483290dbb5b9487c3ce15ca60. Thanks for the review!
